### ISSUE TITLE
Decentralized Identifiers (DID)

### ISSUE THREAD

**Issue by chaals** [w3c/strategy#159]
There is a proposal from a number of members for this. Where does that put it in the funnel?

**Comment by wseltzer** [comment]
Adding to Incubation, with a link to the CCG's [draft spec](https://w3c-ccg.github.io/did-spec) and [draft WG charter](https://w3c-ccg.github.io/did-wg-charter/)

**Comment by koalie** [comment]
[Work in Progress on Decentralized Identifier Working Group Charter (advance notice)](https://lists.w3.org/Archives/Public/public-new-work/2019Feb/0004.html) (sent 2019-02-07)

**Comment by burnburn** [comment]
Name should be "Decentralized Identifiers", not "Distributed Identifiers"

**Comment by chaals** [comment]
@burnburn:

> Name should be "Decentralized Identifiers", not "Distributed Identifiers"

Indeed. Fixed.

**Comment by r12a** [comment]
There is no mention whatsoever in the things i have read so far that anything other than ASCII characters can be used for these identifiers.  Should there be?

**Comment by svgeesus** [comment]
DID seems to draw strongly from URN, so [2.  URN Syntax](https://tools.ietf.org/html/rfc8141#section-2) seems relevant:

> However, this specification does not extend the URN syntax to allow
   direct use of characters outside the ASCII range [RFC20].  That
   restriction implies that any such characters need to be percent-
   encoded as described in Section 2.1 of the URI specification
   [RFC3986].

**Comment by iherman** [comment]
The DID WG has been approved and announced to the AC:

https://lists.w3.org/Archives/Member/w3c-ac-members/2019JulSep/0034.html
